package.json

```diff
@@ -37,6 +37,8 @@
   },
   "author": "Rashid Khan <rashid.khan@elastic.co>",
   "scripts": {
+    "bazel": "bazel",
+    "ibazel": "ibazel",
```

Comment:
This already works because ibazel is available at `node_modules/.bin/ibazel` thanks to `@bazel/ibazel`.

The `bazel` script doesn't exist though, which is surprising...

Reply:
ups yeah you're right, I'll close that PR.